setup.py

```diff
@@ -23,6 +23,7 @@
     entry_points={
         "console_scripts": [
             "dagshub-mount = dagshub.streaming.mount:main",
+            "dagshub-auth = dagshub.auth:console_entrypoint",
```

Comment:
The result command line would be `dagshub-auth`? Any easy option to make it `dagshub login`?